src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -851,6 +851,18 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
                 }
             }
 
+            // apply http configuration configuration
+            configSection = (JObject)config["http"];
+            string routePrefix = ScriptConstants.DefaultHttpRoutePrefix;
+            if (configSection != null)
+            {
+                if (configSection.TryGetValue("routePrefix", out value))
+                {
+                    routePrefix = (string)value;
+                }
```

Comment:
I opened https://github.com/Azure/azure-webjobs-sdk-script/issues/748 to track this and the duplicate route issue. I'll address that in a follow up.